src/components/website-management/website-form-components/customization/CustomizationTabs.js

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
```

Comment:
This is a bad practice as this will import the whole react package. Let's import only the necessary things instead.

```suggestion
import React from 'react';
```

Reply:
fixed